app/renderer/actions/Session.js

```diff
@@ -709,3 +710,18 @@ export function setVisibleProviders () {
     dispatch({type: SET_PROVIDERS, providers});
   };
 }
+
+/**
+ * Check if Safari is started, if so add the includeSafariInWebviews
+ * for future HTML detection
+ *
+ * @param {object} caps
+ */
+function includeSafariInWebviews (caps) {
+  const {browserName = ''} = caps;
+
+  return {
+    ...caps,
+    ...(browserName.toLowerCase() === 'safari' ? {includeSafariInWebviews: true} : {}),
```

Comment:
Is it safe to check 'platformName', too?
Currently, we do not have a plan to add browser safari, but includeSafariInWebviews works only for our mobile safari.

Reply:
I can add that if we want to be on the safe side

Reply:
@KazuCocoa 

Was just thinking about that, but there are some cloud vendors that don't make those capabilities like platformName/version mandatory. That would break this functionality if we would add more caps like that. Can we just leave it like this?

Reply:
ah, okay. Np for current way.
I came up with safaridriver case as an example, but safaridriver should have not work with current capability since newer one only accepted W3C capabilities. So, current AD's behaviour also does not work for them directly.